tools/build-on-netbsd

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+fail() { echo "FAILED:" "$@" 1>&2; exit 1; }
+
+# Check dependencies:
+depschecked=/tmp/c-i.dependencieschecked
+pkgs="
+   bash
+   dmidecode
```

Comment:
speaking of dmidecode! does tools/ds-identify need fixing?

Reply:
I cannot really tell, I don't install `dmidecode` in the image. So I just get some error messages that I ignore.

I've just installed it and the output seems to be rather standard.
[dmidecode.txt](https://github.com/canonical/cloud-init/files/3997266/dmidecode.txt)